WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/ActivityLogListFragment.kt

```diff
@@ -230,10 +223,17 @@ class ActivityLogListFragment : Fragment() {
         viewModel.onActionButtonClicked(item)
     }
 
+    private fun onSecondaryActionClicked(
```

Comment:
1) Idea (💡 ): Since this is going to be the only action later on, once the feature exclusively uses the more menu, what if this is renamed to `onMoreMenuClicked(...)` instead?
2) Idea (💡 ): If the above seems reasonable, then I would suggest renaming every occurrence of `secondary` with `moreMenu`, included in tests. For example, the `secondaryActionClickListener` can become the more explicit `moreMenuClickListener`.
3) Question (❓ ): Why does this click listener returns a `Boolean` value instead of `Unit`? Using `Unit` instead will simplify its usage everywhere. Wdyt?

Reply:
@zwarm Aaa ok then, now I understand how you thought about it, I was thinking it from another angle!

What about then renaming that to `onMenuItemClicked(...)` and a `menuItemClickLIstener` instead?

Reply:
@ParaskP7 - Staying consistent with other places in the app that call menu clicks `SecondaryActions`. :) This sounds like a good conversation to have with the rest of the Raven folks and pick a direction. Thanks for the thoughts.

Reply:
Thanks for letting me know of this Annmarie, didn't know that this is a convention already! So, definitely, staying consistent is the way to go here. 🌟